manual/python/index.md

```diff
@@ -66,9 +66,95 @@ caption="Example of graphics generated with PyROOT."
 
 ## Loading user libraries & jitting
 
-*Content is coming soon!*
+PyROOT allows to use *any C++ library* from Python, not only ROOT. This is possible thanks to the automatic and dynamic bindings between Python and C++ that PyROOT provides: without any prior generation of wrappers, at execution time, PyROOT can load and call into C++ code. The following sub-sections cover different supported scenarios for dynamic C++ code invocation.
```

Comment:
We should point here somewhere to the cppyy docs. Or did you point to them somewhere else?

Reply:
Yes, I point to the cppyy docs at the beginning of the PyROOT section.